USER_MANUAL.md

```diff
@@ -300,7 +307,93 @@ Note that programs are interpreted as a flat list of parameters that are incorpo
+
+**Notes of caution:** A baseline program is required to successfully run the simulation.
+
+### &lt;reference_program&gt; WIP
+
+**Data type:** String
+
+**Default input:** 'P_OGI'
+
+**Description:** Refers to a [name of a program](#program_name), which alternative programs are compared.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** N/A
+
+### &lt;processes_count&gt;
+
+**Data type:** Integer (Numeric)
+
+**Default input:** 6
+
+**Description:** The maximum number of parallel tasks or processes that the simulator can use simultaneously. To simplify, this is the limit of how many different tasks the simulator can handle at a given time.
+
+**Notes on acquisition:** In general, many modern computers can effectively handle around 6 concurrent processes without significant performance issues. This number is influenced by factors such as the computer's hardware specifications, operating system efficiency, and the resource demands of the individual processes.
+
+**Notes of caution:** Python is limited and cannot achieve true simultaneous execution of Python code. Therefore, generating too many processes may cause slower performance due to excessive context switching between the different processes.
+
+A minimum of a single process is required for the simulation to run.
+
+### &lt;simulation_count&gt;
+
+**Data type:** Integer (Numeric)
+
+**Default input:** 2
+
+**Description:** The quantity of simulation iterations to execute for the given programs.
+
+**Notes on acquisition:** Increasing the number of simulations improves result accuracy but extends the runtime.
+
+**Notes of caution:** For critical scenarios that are intended to guide decision-making, we recommend running a significant number of simulations for each modeled scenario. A minimum of 50 simulations is advised to ensure a robust comparison of different LDAR programs.
+
+To generate meaningful and reliable data output from the simulator, it's important to aim for a substantial number of annual data points. We recommend aiming for at least 400 annual data points. This can be achieved by adjusting the number of simulation iterations and the duration of the simulations. For instance, you could run 100 iterations of a 4-year-long simulation. Alternatively, you can employ any combination that suits your specific needs, as long as it results in a sufficient number of data points.
+
+### &lt;preseed_random&gt;
+
+**Data type:** Boolean
+
+**Default input:** False
+
+**Description:** If enabled, a series of random integers, acting as the 'seed', will be generated. These integers guarantee the reproducibility of all randomly generated values. For instance, future simulations utilizing the same virtual world parameters alongside the specified 'seed' will produce identical emission sets.
```

Comment:
Nit: Not sure users need to know how a random seed works. It may be better to simply say something along the lines of setting this to true fixes all random results in the simulation so that they are reproduceable.

Reply:
i think this is fine.